README.md

```diff
@@ -10,6 +10,10 @@ different environments, including:
 *   [Cloud Run and Cloud Run on GKE](https://cloud.google.com/run/)
 *   [Knative](https://github.com/knative/)-based environments
 
+## Video
+
+<a href="https://www.youtube.com/watch?v=qQiqo1zZJmI"><img width="496" alt="Functions Framework" src="https://user-images.githubusercontent.com/744973/71794367-bdafaa80-3073-11ea-8728-61d24942bc56.png"></a>
```

Comment:
I suggest the alt text convey that it's a link to watch a video. As an alternative to the alt text, can also set up link text as a caption and flag that the image should be ignored by assistive tech: https://www.w3.org/TR/WCAG20-TECHS/H67.html

```suggestion
<a href="https://www.youtube.com/watch?v=qQiqo1zZJmI">
<img width="496" alt="" src="https://user-images.githubusercontent.com/744973/71794367-bdafaa80-3073-11ea-8728-61d24942bc56.png">

Watch a video about the Functions Framework</a>
```